### DISCUSSION TITLE
New footnotes style is misaligned

### DISCUSSION THREAD

**Discussion by jab** [pradyunsg/furo#452] [Potential Issue] [answered]
Just tried upgrading to latest furo (2022.6.4.1), Sphinx (5.0.1), and docutils (0.18.1), from versions 2022.4.7, 4.5.0, and 0.17.1 (respectively), as part of a regular maintenance task of running pip-compile-multi. (Last time I did this was June 1, when all went well – a lot changed in a few days:)

From a quick look at the resulting html, I immediately noticed that the new style of footnotes is misaligned:

Before:
<img width="1520" alt="Screen Shot 2022-06-05 at 10 21 49 AM" src="https://user-images.githubusercontent.com/64992/172055197-581e753d-469f-4297-b490-2c39f4e9b0c7.png">

After:
<img width="1520" alt="Screen Shot 2022-06-05 at 10 22 21 AM" src="https://user-images.githubusercontent.com/64992/172055241-ecb1a308-6e98-4274-ab91-482d58fb1a35.png">

Source code is at https://github.com/jab/bidict, and you can run `pip-compile-multi && pip install -r requirements/docs.txt && make -C docs html` in the dev container to reproduce.

Thanks for the great work on Furo!

**Comment by jab** [comment]
(Note, this looks like a different issue from that reported in https://github.com/pradyunsg/furo/discussions/428, which predates the recent changes that are causing this issue by several weeks.)

**Reply by jab** [reply]
ref: https://pradyunsg.me/furo/changelog/#naughty-nickel

> 2022.06.04 – Naughty Nickel[#](https://pradyunsg.me/furo/changelog/#naughty-nickel)
> ...
> When built with docutils 0.18, footnotes are rendered differently and stylised differently in Furo.

**Comment by pradyunsg** [comment, marked as answer]
This is a consequence of a regression/markup change in docutils. See more details in https://github.com/pradyunsg/furo/discussions/447#discussioncomment-2863617.

**Reply by jab** [reply]
Thanks for the quick response! I read the issue you linked to, good to know you were aware of some relevant changes. I don’t see an issue though that tracks any followup work (either to Furo or whatever dependency is responsible). Would you be willing to (create, if necessary, and) link to a followup issue that affected users could track, so we know when it’s safe to try upgrading again?

Also, should corresponding Furo changelog entry more clearly warn users that the footnote style didn’t just change, it actually regressed to a buggy alignment?

**Reply by pradyunsg** [reply]
This is a bug/issue related to docutils -- you can work around this for now, by pinning that to <0.18, since 0.18 release of that package is the where this issue is born out of.

**Reply by pradyunsg** [reply]
I have not reported this to docutils, not least because its development is still undertaken on SourceForge -- https://sourceforge.net/p/docutils/

**Reply by pradyunsg** [reply]
https://github.com/sphinx-doc/sphinx/issues/10531 for Sphinx. That has a link to docutils' issue as well.

**Reply by jab** [reply]
Thank you! Upvoted and subscribed. Great to see someone submitted a patch to https://sourceforge.net/p/docutils/bugs/450/ already. Looking forward to upgrading once fixes make it to release.
